Core/src/org/sleuthkit/autopsy/centralrepository/contentviewer/DataContentViewerOtherCases.java

```diff
@@ -179,17 +181,23 @@ private void showCommonalityDetails() {
             try {
                 EamDb dbManager = EamDb.getInstance();
                 for (CorrelationAttribute eamArtifact : correlationAttributes) {
-                    percentage = dbManager.getFrequencyPercentage(eamArtifact);
-                    msg.append(Bundle.DataContentViewerOtherCases_correlatedArtifacts_byType(percentage,
+                    try {
+                        percentage = dbManager.getFrequencyPercentage(eamArtifact);
+                        msg.append(Bundle.DataContentViewerOtherCases_correlatedArtifacts_byType(percentage,
                             eamArtifact.getCorrelationType().getDisplayName(),
                             eamArtifact.getCorrelationValue()));
+                    } catch (CorrelationAttributeNormalizationException ex) {
+                        String message = String.format("Unable to determine commonality for artifact %s", eamArtifact.toString());
+                        LOGGER.log(Level.SEVERE, message, ex);
+                        Exceptions.printStackTrace(ex);
```

Comment:
It's the getFrequencyPercentage count that fails in this scope so that's why I used that message, but I guess the underlying issue is failed validation.  The message attached to the exception will contain notes on validation failure, and the log message will explain the effects of this failure in this particular case.  This is the approach I've generally tried to take throughout this feature.  Does this seem more reasonable now, or do you still think I should change it?

Reply:
Yeah you can leave that as-is